CMakeLists.txt

```
@@ -246,6 +251,12 @@ if("${DDC_BUILD_KERNELS_SPLINES}")
   find_package(Ginkgo 1.7.0 EXACT REQUIRED)
   target_link_libraries(DDC INTERFACE Ginkgo::ginkgo)
   target_compile_definitions(DDC INTERFACE ginkgo_AVAIL)
+
+  # Lapack
+  find_package(LAPACK REQUIRED)
+  target_link_libraries(DDC INTERFACE LAPACK::LAPACK -llapacke)
+  # The following should not be necessary for LAPACKE >=3.9.1
+  # target_compile_definitions(DDC INTERFACE HAVE_LAPACK_CONFIG_H LAPACK_COMPLEX_CPP)
```

```suggestion
  target_link_libraries(DDC INTERFACE LAPACK::LAPACK -llapacke) # Requires LAPACKE >= 3.9.1
```

The `target_compile_definitions` line is not tested